tests/multi_ptr/multi_ptr_prefetch_member.h

```diff
@@ -0,0 +1,86 @@
+
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides code for multi_ptr prefetch member
+//
+*******************************************************************************/
+
+#ifndef __SYCLCTS_TESTS_MULTI_PTR_PREFETCH_FUNC_H
+#define __SYCLCTS_TESTS_MULTI_PTR_PREFETCH_FUNC_H
+
+#include "../common/common.h"
+
+#include "../common/section_name_builder.h"
+#include "multi_ptr_common.h"
+
+namespace multi_ptr_prefetch_member {
+
+constexpr int expected_val = 42;
+
+/**
+ * @brief Provides functions for verification multi_ptr prefetch member
+ * @tparam T Current data type
+ * @tparam IsDecoratedT sycl::access::decorated enumeration's field
+ */
+template <typename T, typename IsDecoratedT>
+class run_prefetch_test {
+  static constexpr sycl::access::decorated decorated = IsDecoratedT::value;
+  using multi_ptr_t =
+      sycl::multi_ptr<T, sycl::access::address_space::global_space, decorated>;
+
+ public:
+  /**
+   * @param type_name Current data type string representation
+   * @param is_decorated_name Current sycl::access::decorated string
+   *        representation
+   */
+  void operator()(const std::string &type_name,
+                  const std::string &is_decorated_name) {
+    auto queue = sycl_cts::util::get_cts_object::queue();
+    T value = user_def_types::get_init_value_helper<T>(expected_val);
+    SECTION(section_name("Check multi_ptr::prefetch()")
+                .with("T", type_name)
+                .with("address_space", "access::address_space::global_space")
```

Comment:
What about other address spaces? For example `local` and `constant` might be interesting to test.

Reply:
The `muti_ptr::prefetch()` is available only when: `Space == access::address_space::global_space`, that says the [SYCL-2020 spec. Table 94. Member functions of multi_ptr class](https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#:~:text=pointer%2C%20always%20undecorated.-,void%20prefetch(size_t%20numElements)%20const,-Available%20only%20when)